docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+                rx.foreach(
+                    State.users, show_user
+                ),
+            ),
+            variant="surface",
+            size="3",
+        ),
+        graph(),
+    )
+```
+
+One thing you may have noticed about your app is that the graph does not appear initially when you run the app, and that you must add a user to the table for it to first appear. This occurs because the `transform_data` event handler is only called when a user is added to the table. In the next section we will explore a solution to this.
+
+
+## Final Cleanup
+
+### Revisiting app.add_page
+
+At the beginning of this tutorial we mentioned that the `app.add_page` function is required for every Reflex app. This function is used to add a component to a page. 
+
+The `app.add_page` currently looks like this `app.add_page(index)`. We could change the route that the page renders on by setting the `route` prop such as `route="/custom-route"`, thos would change the route to `http://localhost:3000/custom-route` for this page.
+
+We can also set a `title` to be shown in the browser tab and a `description` as shown in search results. 
+
+So solve the problem we had above about our graph not loading when the page loads, we can use `on_load` inside of `app.add_page` to call the `transform_data` event handler when the page loads. This would look like `on_load=State.transform_data`. Below see what our `app.add_page` would look like with some of the changes above added.
+
+```python eval
+rx.vstack(
+    add_customer_button(),
+    rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.foreach(
+                State4.users, show_user
+            ),
+        ),
+        variant="surface",
+        size="3",
+    ),
+    graph(),
+    on_mouse_enter=State4.transform_data
+)
+```
+
+```python
+app.add_page(
+    index,
+    title="Customer Data App",
+    description="A simple app to manage customer data.",
+    on_load=State.transform_data,
+)
+```
+
+### Revisiting app=rx.App()
+
+At the beginning of the tutorial we also mentioned that we defined our app using `app=rx.App()`. We can also pass in some props to the `rx.App` component to customize the app.
+
+The most important one is `theme` which allows you to customize the look and feel of the app. The `theme` prop takes in an `rx.theme` component which has several props that can be set. 
+
+The `radius` prop sets the global radius value for the app that is inherited by all components that have a `radius` prop. It can be overwritten locally for a specific component by manually setting the `radius` prop.
+
+The `accent_color` prop sets the accent color of the app. Check out other options for the accent color [here]({docs.library.other.theme.path}).
```

Comment:
These are a few of the props; where can I see the rest? And what else can I set these props to? Maybe a redirect to some page idk.

Reply:
added link to theming docs